.markdownlint.json

```diff
@@ -1,31 +0,0 @@
-{
-  "default": true,
-  "MD004": false,
-  "MD013": false,
-  "MD022": false,
-  "MD024": false,
-  "MD025": false,
-  "MD026": false,
-  "MD027": false,
-  "MD032": false,
-  "MD033": {
```

Comment:
Are you sure if this should be also removed or kept?

Reply:
Technically this repo could run both, but I think the idea would be to clean them up with one or the other